frontend/assets/stylesheets/components/_quote.scss

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   Promos: Quote
+   ========================================================================== */
+
+.promo-quote {
+    margin-bottom: $gs-baseline * 2;
+}
+
+
+.promo-quote__body {
+    @include clip-path('quote-mobile') {
+        padding-bottom: 168 - $gs-gutter * 2 - $gs-baseline;
+    }
+
+    @include clearfix();
+    background-color: color(neutral-7);
+    padding: $gs-baseline ($gs-gutter / 2) 0;
+    @include mq($from:tablet){
+        @include clip-path('quote') {
+            padding-bottom: 84 - $gs-gutter * 2 - $gs-baseline;
+        }
+    }
+
+}
+
+.promo-quote__text {
+    @include mq($from:tablet){
```

Comment:
you can just write `@include mq(tablet)` as shorthand for this, here and elsewhere